src/index.ts

```diff
@@ -148,7 +148,9 @@ const createProxyHandler = <T extends object>(origObj: T, frozen: boolean) => {
     },
   };
   if (frozen) {
-    handler.set = handler.deleteProperty = () => false;
+    handler.set = handler.deleteProperty = (_target, key) => {
+      throw new Error(`Cannot assign to read only property '${String(key)}'`);
```

Comment:
Open to tweaking this error message; I made it match-ish the normal "you can't write this property" error, but we could also include something about this being a proxy / purposefully make it look less like a built-in error.

Reply:
My preference is to leave it unchanged. What is the error message with returning false?

Reply:
The error is:

```
"'set' on proxy: trap returned falsish for property 'a'"
```

Which I thought was a little too implementation-detail-y; like as a user, I would be like...wait, is this a bug in my code? Valtio? proxy-compare?

My thought was that saying "this is a readonly property" would be more intuitive, b/c that is what would happen if the user was not going through proxy-compare / we had not made a not-frozen-copy to get other non-setter proxy traps to work.

Reply:
I don't disagree that it's not developer friendly, but this should be a rare case.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/set
I think returning `false` is what I want, instead of always throwing.

Reply:
Cool, I've restored the `false`.